doc/README-mosquitto.md

```diff
@@ -0,0 +1,151 @@
+# Ecplipse mosquitto_password cracking
```

Comment:
Typo in `Ecplipse`.

Reply:
Good spot, typo in work 1 of the readme - best avoided :')